homeworks/szabolcsgesztesi/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -40,6 +40,6 @@
 print(f"Végeredmény: Első játékos {player1_score} pont, Második játékos {player2_score} pont")
 if player1_score > player2_score:
     print("Első játékos nyert!")
-else:
+elif player2_score > player1_score:
     print("Második játékos nyert!")
-
+else: print("Döntetlen a játék!")
```

Comment:
Azt ajánlottad, hogy egy while loopot használjak, hogy addig kérje be amíg nem lesz ugyanaz amit a másik játékos választott. Hm, szerintem ez igazából életszerűtlenné teszi a játékot, mert a valóságban is lehet döntetlen.

Így mindent meghagytam ugyanúgy, és csak azt javítottam ami egyébként valóban hiányzott: az eredmények végső kiértékelésénél beraktam egy "else"-t, amit akkor ír ki ha egyik játékos eredménye sem nagyobb mint a másiké, így van megoldás a döntetlen kiírására. :) Így jó lesz?

Reply:
Ezek szerint máshogy játszottuk gyerekkorunkban a kő papír ollót :) Mi sosem álltunk meg döntetlennél, mert pont az volt a lényeg hogy valaki nyerjen :)

De elfogadom a logikádat, a lényeg számomra igazából az, hogy érted hogy a kódot hogyan kellett volna megváltoztatni, nem az hogy a saját elképzelésem ráderőltessem.